rancid-tomatillos/src/Components/Header/Header.test.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Header, mapStateToProps, mapDispatchToProps } from './Header';
+import { removeUser, updateLoggedIn } from '../../actions';
+
+describe('Header', () => {
+  let mockData;
+  beforeEach(() => {
+    mockData = {
+      movies: [{
+        average_rating: 6,
+        backdrop_path: "https://image.tmdb.org/t/p/original//zTxHf9iIOCqRbxvl8W5QYKrsMLq.jpg",
+        id: 1,
+        overview: "In Jumanji: The Next Level",
+        poster_path: "https://image.tmdb.org/t/p/original//l4iknLOenijaB85Zyb5SxH1gGz8.jpg",
+        release_date: "2019-12-04",
+        title: "Jumanji: The Next Level",
+      },
+      {
+        average_rating: 5.666666666666667,
+        backdrop_path: "https://image.tmdb.org/t/p/original//5BwqwxMEjeFtdknRV792Svo0K1v.jpg",
+        id: 2,
+        overview: "The near future",
+        poster_path: "https://image.tmdb.org/t/p/original//xBHvZcjRiWyobQ9kxBhO6B2dtRI.jpg",
+        release_date: "2019-09-17",
+        title: "Ad Astra",
+      }]
+    };
+  });
+
+  describe('Header Component', () => {
+    let wrapper;
+    let mockRemoveUser;
+    let mockUpdateLoggedIn;
+    let mockPath;
+
+    beforeEach(() => {
+      const mockMath = Object.create(global.Math);
+      mockMath.random = () => .05;
+      global.Math = mockMath;
+      mockRemoveUser = jest.fn();
+      mockUpdateLoggedIn = jest.fn();
+    });
+
+    it('should match the snapshot with isLoggedIn as true and path not containing movies', () => {
```

Comment:
Ooo that mockMath implementation looks like fun, you will have to tell us about it when we meet tomorrow